docs/topics/testing/tools.txt

```diff
@@ -158,25 +158,28 @@ Use the ``django.test.Client`` class to make requests.
     Once you have a ``Client`` instance, you can call any of the following
     methods:
 
-    .. method:: Client.get(path, data=None, follow=False, secure=False, *, headers=None, **extra)
+    .. method:: Client.get(path, data=None, follow=False, secure=False, *, headers=None, query_params=None, **extra)
```

Comment:
`versionchanged` annotations are missing for every method.

Reply:
`query_params=None` is missing in `AsyncClient` and `Client` signatures and docs.